services/document-manager/backend/migrations/versions/9be7dbc60ff8_add_status_message_to_document_table.py

```diff
@@ -0,0 +1,26 @@
+"""add_status_message_to_document_table
+
+Revision ID: 9be7dbc60ff8
+Revises: c02786c20bfd
+Create Date: 2023-09-08 21:32:57.125152
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+# revision identifiers, used by Alembic.
+revision = '9be7dbc60ff8'
+down_revision = 'c02786c20bfd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("document") as batch_op:
+        batch_op.add_column(sa.Column('status', sa.String(length=255), server_default=text("'In Progress'"), nullable=True))
```

Comment:
Will this mark all existing documents as "In Progress"?

Thinking it might be better to maybe lave it blank or set it to `Success` for existing ones to avoid any future confusion 😅

Reply:
Yeah it will.  Good point that it might make it confusing.  I think I'll set existing ones as Success and keep the default for future ones.